packages/logger/src/interfaces/ogma-options.ts

```diff
@@ -39,6 +39,18 @@ export interface OgmaOptions {
    * @default ''
    */
   application: string;
+  /**
+   * A value to say if you want to log pid or not
+   */
+  logPid?: boolean;
+  /**
+   * A value to say if you want to log application name or not
+   */
+  logApplication?: boolean;
+  /**
+   * A value to say if you want to log hostname or not
+   */
+  logHostname?: boolean;
```

Comment:
Just had a thought here: could we put this under something like `customizeOutput` in a setup like

```
/**
 * how the logger output should be customized
 * /
customizeOutput?: {
  pid?: boolean,
  application?: boolean;
  hostname?: boolean;
}
```

So that later we can do a surrounding if check of `if (options.customizeOutput)` and then check all the sub-fields as necessary? Should cut down the if checks if we don't want to customize anything as we'd fallback to the standard output I've already created. 

Thoughts on the approach?

Reply:
Maybe it doesn't add too much value with us using a default `true` value for the `logX` properties, so if you think this will add in too much confusion / not a good dev XP then I'm fine to scrap the idea as well.

Reply:
that approach will make fewer IF checks in case there are no modifications, but IF check is cheap. I'd rather stick to original idea (don't see good reasons to do a switch for now)